src/components/WhoWeAre.tsx

```diff
@@ -2,84 +2,71 @@ import { ReactElement } from 'react';
 
 import { cn } from '@/lib/utils';
 
-import NextImage from '@/components/NextImage';
 import NextLink from '@/components/NextLink';
 
+import GitHubLogo from '~/svg/github.svg';
+import LinkedInLogo from '~/svg/linkedin.svg';
+import KenoLogo from '~/svg/who-we-are-keno.svg';
+import PhilippLogo from '~/svg/who-we-are-philipp.svg';
+import TimoLogo from '~/svg/who-we-are-timo.svg';
+import TomLogo from '~/svg/who-we-are-tom.svg';
+
 function ExternalReferences({
   linkedInLink,
   gitHubLink,
-  size,
   className,
+  classNameLogo,
 }: {
   linkedInLink: string;
   gitHubLink: string;
-  size: number;
-  className?: string;
+  className: string;
+  classNameLogo: string;
 }) {
   return (
     <div className={cn('gap-1', className)}>
       <NextLink href={linkedInLink} isExternalLink>
-        <NextImage
-          src='/svg/linkedin.svg'
-          alt='linkedin-logo'
-          width={size}
-          height={size}
-          useSkeleton
-        />
+        <LinkedInLogo className={cn(classNameLogo)} />
       </NextLink>
       <NextLink href={gitHubLink} isExternalLink>
-        <NextImage
-          src='/svg/github.svg'
-          alt='github-logo'
-          width={size}
-          height={size}
-          useSkeleton
-        />
+        <GitHubLogo className={cn(classNameLogo)} />
       </NextLink>
     </div>
   );
 }
 
 function TeamMemberField({
-  imgSource,
+  logo,
   name,
   description,
   linkedInLink,
   gitHubLink,
 }: {
-  imgSource: string;
+  logo: ReactElement;
   name: string;
   description: ReactElement;
   linkedInLink: string;
   gitHubLink: string;
 }) {
   return (
     <div className='flex items-start justify-between gap-5 lg:items-center lg:justify-start lg:gap-16'>
-      <NextImage
-        src={imgSource}
-        alt='logo'
-        width={85}
-        height={0}
-        className='hidden lg:inline-flex'
-        useSkeleton
-      />
+      {logo}
```

Comment:
warum hast du das hier rausgenommen?

Reply:
Durch den SVGR change gebe ich jetzt nicht mehr die URL mit sondern die Logo Component selbst. Allerdings sollte ich das Logo auch als ComponentType reingeben können, dann kann ich die classNames in der TeamMemberField Component lassen. Passe ich noch an!